Puppetfile

```diff
@@ -1,7 +1,7 @@
 forge 'https://forgeapi.puppetlabs.com'
 
 mod 'atsonkov/grubby', '0.3.2'
-mod 'blockops/tailscale', git: 'https://github.com/lsst-it/puppet-tailscale', ref: 'd06b499'  # 1.0.0 + https://gitlab.com/blockops/puppet-tailscale/-/merge_requests/13
+mod 'blockops/tailscale', git: 'https://github.com/lsst-it/puppet-tailscale', ref: '9da42cd'  # 1.0.0 + https://gitlab.com/blockops/puppet-tailscale/-/merge_requests/13
```

Comment:
The comment needs to be updated and the branch the ref is on need to be rebased on upstream too